app/src/main/java/org/fossasia/openevent/app/login/LoginPresenter.java

```diff
@@ -1,26 +1,36 @@
 package org.fossasia.openevent.app.login;
 
 import org.fossasia.openevent.app.data.contract.ILoginModel;
+import org.fossasia.openevent.app.data.contract.IUtilModel;
 import org.fossasia.openevent.app.login.contract.ILoginPresenter;
 import org.fossasia.openevent.app.login.contract.ILoginView;
+import org.fossasia.openevent.app.utils.Constants;
+
+import java.util.Set;
 
 import javax.inject.Inject;
 
 public class LoginPresenter implements ILoginPresenter {
 
     private ILoginView loginView;
     private ILoginModel loginModel;
+    private IUtilModel utilModel;
 
     @Inject
-    public LoginPresenter(ILoginModel loginModel) {
+    public LoginPresenter(ILoginModel loginModel, IUtilModel utilModel) {
         this.loginModel = loginModel;
+        this.utilModel = utilModel;
     }
 
     @Override
     public void attach(ILoginView loginView) {
         this.loginView = loginView;
+
         if(loginView != null && loginModel.isLoggedIn())
             loginView.onLoginSuccess();
+
+        if(loginView != null && getEmailList() != null)
```

Comment:
These lines are adjacent. So, move null check on top and return if it is null or logged in and only check for email list being null before attaching them